grammar.js

```diff
@@ -167,7 +169,8 @@ module.exports = grammar({
             field('parameters', alias($.bare_parameters, $.method_parameters))
           ),
           $._terminator,
-          $._body_statement
+          optional(field("method_body", $.body)),
```

Comment:
A minor nit: could you please use `'` instead of `"`. The rest of the file seems to use single quotes, so let's keep things consistent.